datalake/backend/endpoints/datasets/create.py

```diff
@@ -2,8 +2,7 @@
 
 from jsonschema import ValidationError, validate  # type: ignore[import]
 
-from ..utils import JSON_OBJECT, error_response, success_response
-from .common import DATASET_TYPES
```

Comment:
I think it would be good to extract this change to separate PR. I like it, but it is out of scope of this PR.